src/components/login/login.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import "./login.scss";
+import { toast } from "react-toastify";
+// import { useSelector } from "react-redux";
```

Comment:
- [ ] In the last review, you asked me to point out the commented code.  Kindly remove this unused commented-out import 👍